tests/test_translation_level_10.py

```diff
@@ -0,0 +1,54 @@
+import hedy
+from test_level_01 import HedyTester
+import hedy_translation
+from test_translating import check_local_lang_bool
+
+# tests should be ordered as follows:
+# * Translation from English to Dutch
+# * Translation from Dutch to English
+# * Translation to several languages
+# * Error handling
+
+
+class TestsTranslationLevel10(HedyTester):
+    level = 10
+
+    @check_local_lang_bool
+    def test_for_english_dutch(self):
+        code = "dieren is hond, kat, vis\n" \
+               "for dier in dieren\n" \
+               "    print 'hallo' dier"
+
+        result = hedy_translation.translate_keywords(code, from_lang="en", to_lang="nl", level=self.level)
+        expected = "dieren is hond, kat, vis\n" \
```

Comment:
Shall we use `textwrap.dedent` here (like the regular tests do)? That reads a bit nicer without the \

Reply:
@Felienne Yes, we forgot the regular tests use that. We can update tests to use textwrap.dedent

Reply:
Thanks, please do (for future levels also, of course)